packages/discussions/src/channels.ts

```diff
@@ -79,3 +86,68 @@ export function removeChannel(
   options.httpMethod = "DELETE";
   return request(`/channels/${options.channelId}`, options);
 }
+
+/**
+ * get channel opt out status
+ *
+ * @export
+ * @param {IFetchOptOutOptions} options
+ * @return {*}
+ */
+export function fetchOptOut(
```

Comment:
Similar, these method names are a little generic. Lets be more explicit to avoid future collisions since.

```
fetchChannelNotifcationOptOut
createChannelNotificationOptOut
removeChannelNotificationOptOut
removeChannelActivity
```